server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,145 @@
+
+* <<_install_by_olm,The Operator Lifecycle Manager (OLM)>> 
+* <<_install_by_command,Command line installation>> 
+
+[[_install_by_olm]]
+==== Installing using Operator Lifecycle Manager
+
+ifeval::[{project_community}==true]
+You can install the Operator on a OpenShift or Kubernetes cluster
+
+===== Installation on an OpenShift cluster
+endif::[]
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
+
+.Procedure
+
+Perform this procedure on an OpenShift 4.3 cluster.
+
+. Open the OpenShift Container Platform web console.
+
+. In the left column, click `Operators, OperatorHub`.
+
+. Search for {project_name} Operator.
++
+.OperatorHub tab in OpenShift
+image:{project_images}/operator-openshift-operatorhub.png[]
+
+. Click the {project_name} Operator icon.
++
+An Install page opens.
++
+.Operator Install page on OpenShift
+image:{project_images}/operator-olm-installation.png[]
+
+. Click `Install`.
+
+. Select a namespace and click Subscribe.
++
+.Namespace selection in OpenShift
+image:images/installed-namespace.png[]
++
+The Operator starts installing.
+
+When the Operator installation completes, you are ready to create custom resources to install {project_name}, create realms, and perform other adminstrative tasks.
+
+ifeval::[{project_community}==true]
+
+===== Installation on a Kubernetes cluster
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
+
+.Procedure
+
+For a Kubernetes cluster, perform these steps.
+
+. Go to link:https://operatorhub.io/operator/keycloak-operator[Keycloak Operator on OperatorHub.io].
+
+. Click `Install`.
+
+. Follow the instructions on the screen.
++
+.Operator Install page on Kubernetes
+image:{project_images}/operator-operatorhub-install.png[]
+
+When the Operator installation completes, you are ready to create custom resources to install {project_name}, create realms, and perform other adminstrative tasks.
+endif::[]
+
+.Additional Resources
+
+ifeval::[{project_community}==true]
+* For more information on a Kubernetes installation, see link:https://operatorhub.io/how-to-install-an-operator[How to install an Operator from OperatorHub.io].
+endif::[]
+
+* For more information on OpenShift Operators, see the link:https://docs.openshift.com/container-platform/4.2/operators/olm-what-operators-are.html[OpenShift Operators guide].
+
+* For more information on OpenShift installation, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html/installing/indexl[Installing and Configuring OpenShift Container Platform clusters].
+
+[[_install_by_command]]
+==== Installing from the command line
+
+You can install the {project_operator} from the command line.
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
```

Comment:
cluster-admin permission

Reply:
@andymunro I do not know is you observed this comment. So I am just assigning it to you. Thank you for your help. :-)